### PR TITLE
Change HEROKU_SLUG_COMMIT  to HEROKU_RELEASE_CREATED_AT 

### PR DESCRIPTION
**Pull request by Ridhwana**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Change related to https://github.com/forem/forem/pull/12149.

RELEASE_FOOTPRINT is usually set in the Forem Cloud env environment. However, we need to cater for those Forems that are being deployed on Heroku as well. 

RELEASE_FOOTPRINT is the datetime that the deploy is done, however the fallback variable for Heroku was set to HEROKU_SLUG_COMMIT. This causes the Last deployed time to be a commit id instead of a date when RELEASE_FOOTPRINT is not set (like on heroku envs). See the image below: 

 Screenshot 2021-01-08 at 17 35 39

Hence, I've changed the fallback to HEROKU_RELEASE_CREATED_AT. 

## Related Tickets & Documents
https://github.com/forem/forem/pull/12149

## QA Instructions, Screenshots, Recordings
1. In the .env file change the following variables as follows:
```
export HEROKU_RELEASE_CREATED_AT="Some Date"
export HEROKU_SLUG_COMMIT="Some Commit ID"
```

2. Now restart your server and go to http://localhost:3000/admin, you should see the correct fallback variable set.
Screenshot 2021-01-08 at 17 44 12

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added tests?

- [ ] Yes
- [x] No, because its a fallback env variable change 
- [ ] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by rhymes — APPROVED** [review]
LGTM!

Though I wonder if we can't just remove this initializer and access the correct variable directly

**Comment by Ridhwana** [comment]
> Though I wonder if we can't just remove this initializer and access the correct variable directly

@rhymes possibly - I do feel like this code can be cleaned up. I'm going to look into how we can clean it up in the follow up PR together with Micheal's suggested change of the ForemInstance.

**Review by jdoss — APPROVED** [review]

**Review by benhalpern — APPROVED** [review]

File: config/initializers/set_release_footprint.rb [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # We will set RELEASE_FOOTPRINT in our Forem Cloud environment, or use HEROKU_SLUG_COMMIT if set (e.g. Heroku env)
```
Comment (maestromac) [inline config/initializers/set_release_footprint.rb:1]:
```suggestion
# We will set RELEASE_FOOTPRINT in our Forem Cloud environment, or use HEROKU_RELEASE_CREATED_AT if set (e.g. Heroku env)
```

Might want to update this comment too

**Review by maestromac — APPROVED** [review]
LGTM!

**Pull request merged**